### PR TITLE
README: primitive attempt to add readme to this repo

### PR DESCRIPTION
**Pull request by mcharanrm**
#### Highlights of this PR 

Created Readme to support the following topics
1. Launch plain IBM cloud instances
2. Launch IBM cloud instances + Install Tower
3. Destroy IBM cloud instances

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,170 @@
...
+We believe there are two use cases associated with this repo and these are our primitive attempts to describe it.
+1. Looking for virtual cloud instances (OR)
+2. Looking for Tower deployment on faster disks
+
+Looking for virtual cloud instances ??
+------------------------------------
+
+Install Ansible modules collection for IBM cloud
+
+    ansible-galaxy collection install awx.awx
```
Comment (jainnikhil30) [inline README.md:15]: it should be:

ansible-galaxy collection install ibm.cloudcollection

Comment (mcharanrm) [reply README.md:15]: my bad. thanks

File: README.md [hunk 2]
```diff
@@ -1 +1,170 @@
...
+```
+[cmusali@cmusali ibm_cloud_tower_deploy]$ cat > ic_instance_inventory.ini
+[ic_servers]
+ic1 ansible_host=169.63.178.143
+ic2 ansible_host=169.59.164.19
+
+
+[ic_servers:vars]
+ansible_user = 'root'          
+ansible_ssh_private_key_file=conf/towerperf_id_rsa
```
Comment (jainnikhil30) [inline README.md:150]: Should we mention that this private key is part of towerperf repo? Also may be we should add a section wherein we tell how to change the ssh key. Basically you need to either pass it via extra vars or change in it vars.yml ?

**Comment by jainnikhil30** [comment]
Thanks a lot @mcharanrm 
Looks good to me with those small changes I suggested above.

File: README.md (hunk 2 above)
Comment (mcharanrm) [reply README.md:150]: >Should we mention that this private key is part of towerperf repo?

It is an attempt to show what can be added to the inventory. 

>Also may be we should add a section wherein we tell how to change the ssh key. Basically you need to either pass it via extra vars or change in it vars.yml ?

Line 113-120 explains `vars.yaml` default config values.

Comment (jainnikhil30) [reply README.md:150]: Perfect. Looks good then.

**Review by jainnikhil30 — APPROVED** [review]

**Pull request merged**